src/client/Microsoft.Identity.Client/Properties/InternalsVisibleTo.cs

```diff
@@ -6,6 +6,7 @@
 
 [assembly: InternalsVisibleTo("Microsoft.Identity.Client.Desktop" + KeyTokens.MSAL)]
 [assembly: InternalsVisibleTo("Microsoft.Identity.Client.Broker" + KeyTokens.MSAL)]
+[assembly: InternalsVisibleTo("Microsoft.Identity.Client.SsoPolicy" + KeyTokens.MSAL)]
```

Comment:
Oh, interesting. You want to create a new DLL / package for this? Is there a reason why you can't reuse the "Broker" DLL / package for this ?